### PR TITLE
Hyperspectral minor fixes to get these running again

### PR DESCRIPTION
**Pull request by donal-at-NEON**
@cklunch please review these changes.

Mostly minor changes to file names and working directory paths. Also, several small edits for grammar (e.g. change 'earth' to 'Earth' (as it is a proper noun) and 'multiple' to 'multiply', etc.). Currently, these tutorials don't 'work' (see the frank and fair comment from CrHvG), so this will at least patch those issues well enough to work properly for now.

Please take a look at the way that I changed the working directory definition, and how I used the paste0() function throughout. I am not sure if this is the preferred style for our tutorials, but I do remember seeing the working directory and other file paths hard-coded in other tutorials so I am guessing this is at least passable for now.

There are still bigger picture things that I want to change - in particular, there are a few figures in the 'Intro to working with hyperspectral' tutorial that suggest that the visible EM spectrum extends to 800nm, when in reality it's closer to 700nm (as we discussed in my office). While I think that is a lower priority, it should still be addressed eventually. I would like to bring in a graphic designer to make those changes to the images (or perhaps find the original image design files in Adobe or something?) as graphic design is way outside of my expertise. 

As always, I will be happy to edit and revise these as needed. Thanks, Claire!

### CODE REVIEW

**Comment by cklunch** [comment]
Thanks Donal! @mjones01 can you review this one? I'm not as familiar with this set of tutorials.

File: tutorials/hyperspectral/Work-With-Hyperspectral-Data-In-R.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ preferably, RStudio loaded on your computer.
 The electromagnetic spectrum is composed of thousands of bands representing 
 different types of light energy. Imaging spectrometers (instruments that collect 
 hyperspectral data) break the electromagnetic spectrum into groups of bands that 
-support classification of objects by their spectral properties on the earth's 
+support classification of objects by their spectral properties on the Earth's
```
Comment (mjones01) [inline tutorials/hyperspectral/Work-With-Hyperspectral-Data-In-R.md]: I'm confused by several changes in the .md that don't appear as corresponding changes in the .Rmd.  But the Review layout is making it challenging to track these down.  I just want to make sure any change in the .md is also in the .Rmd.

File: tutorials/hyperspectral/click_points_rough.R [hunk 2]
```diff
@@ -0,0 +1,70 @@
+
```
Comment (mjones01) [inline tutorials/hyperspectral/click_points_rough.R]: I think these must be scrips where you're testing out items?  We don't want to pull these into the main repo, I don't think.  Can they be moved to a branch or just stored locally?

File: tutorials/hyperspectral/high_res_rough.R [hunk 3]
```diff
@@ -0,0 +1,19 @@
+wd="~/Desktop/Hyperspectral_Tutorial/" #This will depend on your local environment
```
Comment (mjones01) [inline tutorials/hyperspectral/high_res_rough.R]: Same as above - seems to below on local or branch, not in main repo?

**Pull request closed without merging**